shared/test/localization/format-test.js

```diff
@@ -172,7 +170,7 @@ describe('format', () => {
 
   describe('#registerLocaleData', () => {
     // Store original locale data so we can restore at the end
-    const moduleLocales = require('../../localization/locales.js');
+    const moduleLocales = locales;
```

Comment:
Should this be doing a deep copy?

Reply:
It should just be deleted and locales used directly in the next line.

Reply:
Ope totally missed the next line. In that case is having `moduleLocales` and `locales` necessary?

Reply:
No, it wasn't even necessary before.